sun/sun-ui/src/services/util.js

```diff
@@ -87,6 +87,12 @@ var UtilService = {
         case 'voucher_login':
           icon = 'fa fa-wpforms login-pref-option'
           break;
+	case 'CoovaChilli-Bandwidth-Max-Down':
```

Comment:
Please fix the indentation.
There are other few lines like this in the patch.

Reply:
ok, done.